WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/promptslist/BloggingPromptsListActivity.kt

```diff
@@ -22,10 +27,11 @@ class BloggingPromptsListActivity : LocaleAwareActivity() {
         setContent {
             AppTheme {
                 val uiState by viewModel.uiStateFlow.collectAsState()
-                BloggingPromptsListScreen(uiState, ::onBackPressed)
+                BloggingPromptsListScreen(uiState, ::onBackPressed) { item -> viewModel.onPromptListItemClicked(item) }
```

Comment:
Suggestion: passing a reference to a method has some [performance advantages](https://multithreaded.stitchfix.com/blog/2022/08/05/jetpack-compose-recomposition/) in Compose, so I suggest doing something like this:

```suggestion
                BloggingPromptsListScreen(
                    uiState,
                    ::onBackPressed,
                    viewModel::onPromptListItemClicked
                )
```